samples/nrf9160/nrf_cloud_rest_device_message_tol2/src/main.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Author: tol2@nordicsemi.no
+ * Date: 07/28/2022
+ * NRFCloud rest message sample app replicate
+ */
+
+#include <zephyr/kernel.h>
+#include <nrf_modem_at.h>
+#include <zephyr/settings/settings.h>
+#include <net/nrf_cloud.h>
+#include <net/nrf_cloud_rest.h>
+#include <zephyr/logging/log.h>
+#include <date_time.h>
+
+LOG_MODULE_REGISTER(nrf_cloud_rest_device_message_sample,
+		    CONFIG_NRF_CLOUD_REST_DEVICE_MESSAGE_SAMPLE_LOG_LEVEL);
+
+#define WAIT_TIME_BETWEEN_MESSAGES_SEC 3
+#define TIME_TO_RETRY 5
+#define MAX_TIME_OUT 60000
+#define REST_RX_BUF_SIZE 2100
+
+char device_id[NRF_CLOUD_CLIENT_ID_MAX_LEN + 1];
+K_SEM_DEFINE(lte_connect, 0, 1);
+K_SEM_DEFINE(modem_time, 0, 1);
+char rx_buf[REST_RX_BUF_SIZE];
+struct nrf_cloud_rest_context rest_context = {
+						   .connect_socket = -1,
```

Comment:
`.connect_socket` does not look to be aligned with the other elements -- it should be aligned

Reply:
Ohh I forgot to edit the code after using clang-format, it's a weird alignment every time I used.